wrapper/src/main/java/software/amazon/jdbc/hostlistprovider/monitoring/AuroraGlobalDbMonitoringHostListProvider.java

```diff
@@ -1,111 +0,0 @@
-/*
```

Comment:
I renamed this to GlobalAuroraMonitoringHostListProvider to be consistent with GlobalAuroraMysqlDialect